metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/sum.rb

```diff
@@ -15,7 +15,7 @@ class Sum
 
           def initialize(aggregation_temporality: ENV.fetch('OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE', :delta))
             # TODO: the default should be :cumulative, see issue #1555
-            @aggregation_temporality = aggregation_temporality
+            @aggregation_temporality = aggregation_temporality.to_sym
```

Comment:
Can this be user specified or is there a constraint on the temporality preference?

I.e. should this be open ended or should it be a sort of enum object?

Reply:
Great point! I think an enum would be a good addition here. 

[According to the spec](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#temporality), the only valid values here are `:delta` or `:cumulative`. The OTLP exporter will fall back to `unspecified` otherwise. 

We don't have support for `:cumulative` aggregation yet, so really, the only true option should just be `:delta`.

Reply:
I opened an issue to fix this as a follow-up: https://github.com/open-telemetry/opentelemetry-ruby/issues/1742